app/databrowser/src/main/java/org/csstudio/trends/databrowser3/model/PVItem.java

```diff
@@ -85,6 +85,10 @@ public class PVItem extends ModelItem
      * the live buffer is too small to show all the data */
     private boolean automaticRefresh = Preferences.automatic_history_refresh;
 
+    /** Indicates whether the display name has been updated for the first time.
+     * once upon the first reception of a value from the PV*/
+    private boolean checkDisplayName = false;
```

Comment:
At this time, probably not, because changes to the description are very rare. I'd say it's acceptable to require a restart of the data browser to see updated descriptions. But maybe add a line to the checkDisplayName comment that we only check once and ignore runtime changes on purpose

Reply:
I've run a few tests to see if it's possible to update the description if it's modified with the PV. 

The problem is that `display.getDescription()` doesn't change even if the `PV.DESC` is modified.

Reply:
Thanks for checking. Might be that QSRV/pvAccessCPP/... as used in EPICS base right now don't support it, while the new C++ implementation in  PVXS/softIocPVX already does or eventually will. So just add a comment that we don't support runtime changes for performance reasons (and at this time the server seems to do the same)